model/itemTab.h

```diff
@@ -48,7 +48,7 @@ namespace minsky
     /// computed positions of the variable rows
     std::vector<double> rowTopMargin;                      
 
-    void populateItemVector();
+    virtual void populateItemVector();
```

Comment:
Don't forget to add a virtual destructor. It will be virtual anyway, because CairoSurface has a virtual destructor, but such things are easily overlooked if ever the inheritance hierarchy is changed,

Reply:
Haven't I already?  See   virtual ~ItemTab() {} on line 79